src/Zed/Persistence/Repository/AbstractRepositoryRule.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace ArchitectureSniffer\Zed\Persistence\Repository;
+
+use ArchitectureSniffer\SprykerAbstractRule;
+use PHPMD\Node\MethodNode;
+
+abstract class AbstractRepositoryRule extends SprykerAbstractRule
+{
+    public const PATTERN_FACTORY = '/\\\\*\\\\.+\\\\.+\\\\[A-Za-z0-9]+Repository$/';
```

Comment:
Copy paste error? Shouldn't it be REPOSITORY_PATTERN?

Reply:
PATTERN_REPOSITORY :) yeah